mods/ra/maps/soviet-06b/soviet06b-AI.lua

```diff
@@ -68,15 +69,18 @@ ProduceArmor = function(factory)
 	end)
 end
 
-ProduceHarvester = function(factory)
+ProduceHarvester = function(factory, delay)
 	if GreeceMoney() < Actor.Cost("harv") then
 		return
 	end
 
 	local toBuild = { "harv" }
-	Greece.Build(toBuild, function(unit)
-		unit.FindResources()
-		ProduceArmor(factory)
+	Greece.Build(toBuild, function(units)
+		units[1].FindResources()
```

Comment:
This call is completelly unnecesary. Harvesters find resources when built by default